tests/strategies/test_abc.py

```diff
@@ -92,6 +93,15 @@ def test_get(
             return_json=testdata(strategy_name),
         )
 
```

Comment:
Maybe add a note why we need the extra logic? Not sure, since its duplicated it might just clog things up...

Reply:
Where should I add the note? Maybe next to all the extra `if`-conditions? Yeah :) 👍

Reply:
I've extended the existing comment in all the `if`-conditions to: `# Mock URL responses to ensure we don't hit the real (external) URL`